### PR TITLE
Allow for env variables for configuring delete-message undo time limit for toasts

### LINKED ISSUE #19644: Increase time of Undo button to be shown when deleting a message
# Problem

One of the cases that we covered by automation in our e2e tests is deleting a message in a chat, undoing that action and checking that the message is restored. This test was stable for a long time, the functionality worked as expected but recently it started failing because of insufficient time for clicking Undo button. Actually we started facing this failure in and after #19270 merged. To have an ability to continue checking this important feature we need a little bit longer timeout for this button to be shown. It can be increased only in e2e apk leaving other builds as it is.

## Implementation

Increase time when the toast with Undo button is shown in e2e builds

## Acceptance Criteria

Undo button is shown and clickable for 8-10 sec

## Steps to test

1. Send a message in a chat
2. Delete the message
3. Check the time while Undo button is present

### PR DESCRIPTION
**Pull request by seanstrom**
[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

fixes #19644 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)

* This PR attempts to the status-mobile to be configured with environment variables for configuring the time limit for displaying the undo toast after deleting a message.
* This PR adds two environment variables: `DELETE_MESSAGE_FOR_ME_UNDO_TIME_LIMIT` and `DELETE_MESSAGE_UNDO_TIME_LIMIT`.
  * `DELETE_MESSAGE_FOR_ME_UNDO_TIME_LIMIT` applies to deleting a message for the user in a chat
  * `DELETE_MESSAGE_UNDO_TIME_LIMIT` applies to deleting a message for everyone in a chat
* To use these environment variables we can define them inside the `.env` files for the status-mobile project. If we want to define these only for E2E tests, then we can define them inside the `.env.e2e` file inside the status-mobile project.

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats

### Steps to test

- Define the environment variables inside the `.env` or `.env.e2e` files for testing.
- If you're building the application, then make sure to run `make clean` before building.
- Open the Status mobile app
- Open a 1-1 chat
- Send a message inside a 1-1 chat
- Long press on that message to open the action menu
- Choose "Delete for me" action or "Delete for everyone" action

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]
we also need to add DELETE_MESSAGE_FOR_ME_UNDO_TIME_LIMIT value 10000 to .env.e2e in this PR

File: .env.e2e [hunk 1]
```diff
@@ -35,3 +35,4 @@ LOCAL_PAIRING_ENABLED=1
 FAST_CREATE_COMMUNITY_ENABLED=1
 TEST_NETWORKS_ENABLED=1
 SHOW_NOT_IMPLEMENTED_FEATURES=1
+DELETE_MESSAGE_FOR_ME_UNDO_TIME_LIMIT=10000
```
Comment (flexsurfer) [inline .env.e2e:38]: DELETE_MESSAGE_UNDO_TIME_LIMIT as well :)

**Review by Parveshdhull — APPROVED** [review]

**Comment by qoqobolo** [comment]
Hey @seanstrom, thanks for the PR!
Can you merge it, please?

**Comment by seanstrom** [comment]
@qoqobolo yup yup, just merged 🙌

**Pull request merged**